### ISSUE TITLE
Warning while using the Maven-Plugin

### ISSUE THREAD

**Issue by larsgrefer** [darrinholst/sass-java#14]
I am using the maven-plugin (version 3.4.4.2) and getting the following warning:
```
Compass.compiler is deprecated. Use Compass.sass_compiler instead.
Called from <script>:2:in `(root)'
```

this is my configuration:
```xml
<plugin>
	<groupId>com.darrinholst</groupId>
	<artifactId>sass-java-maven-plugin</artifactId>
	<version>3.4.4.2</version>
	<executions>
		<execution>
			<goals>
				<goal>compile</goal>
			</goals>
			<configuration>
				<config>${project.build.directory}/${project.build.finalName}/config.rb</config>
			</configuration>
		</execution>
	</executions>
</plugin>
```

**Comment by brianlenz** [comment]
Here's all that I think should be needed to switch to using the new SassCompiler (sass_compiler). In buildCompileScript(), replace the current Compass.compiler line with:

```script.println("  compiler = Compass.sass_compiler");
        script.println("  compiler.logger = Compass::NullLogger.new");
        script.println("  compiler.clean!");
        script.println("  compiler.compile!");```

That will also prevent it from doing the log output to standard out.

Then in buildInitializationScript(), add the following after the compass require:

```script.println("require \'compass/sass_compiler\'                                                          ");```

That gets it using the new SassCompiler. Works great in my testing...

**Comment by darrinholst** [comment]
Thanks @brianlenz and @larsgrefer. I've released version 3.4.4.3 with the changes above.

**Comment by brianlenz** [comment]
Glad to have been able to help! Thanks for the great library :+1:
